src/renderer/Generics/redesign/LogMessage/LogMessage.tsx

```diff
@@ -24,10 +24,6 @@ export interface LogMessageProps {
 }
 
 export const LogMessage = ({ timestamp, level, message }: LogMessageProps) => {
-  const copy = async () => {
-    await navigator.clipboard.writeText(message);
-  };
-
   return (
     <div className={container}>
```

Comment:
One minor thing that looks like it might be missing is the `background-color` change when a LogMessage is hovered